code/game/objects/structures/orbital_cannon.dm

```diff
@@ -365,7 +365,7 @@
 	icon_state = "ob_warhead_1"
 
 /obj/structure/ob_ammo/warhead/explosive/warhead_impact(turf/target, inaccuracy_amt = 0)
-	explosion(target, 5 - inaccuracy_amt, 7 - inaccuracy_amt, 10 - inaccuracy_amt, 11 - inaccuracy_amt)
```

Comment:
Just to be clear, that is 6 tiles of insta gib for like 70% of xenos.
Just checking the intention is clear.

The rest im good with 👍

Reply:
If they decide to stand in the OB laser after it left or hear it beep loudly and stay in range for a OB then cluster would've have killed them 10 times over. HE before was basically a HE shell for mortars but the value for heavy 7 is supposed to be 6 i'll go fix that